src/aiai_eval/model_adjustment.py

```diff
@@ -136,16 +103,14 @@ def adjust_model_to_task(
         }
 
         # Get the old model id2label conversion
-        old_model_id2label = [
-            model.config.id2label[idx].upper()
-            for idx in range(len(model.config.id2label))
-        ]
+        old_model_id2label = model_config.id2label
 
         # Alter the model's classification layer to match the dataset if the model is
-        # missing labels
+        # missing labels. This only works if the model is a Hugging Face PyTorch model
```

Comment:
I opened an issue addressing this point: #73 . For now we could log a snippet at this point that one should be wary with local models?

Reply:
Hmm. If a user uses a local model then they'd simply encounter an error if the model doesn't have the correct amount labels. I suppose I can add an informative error if they supply fewer labels than what is in the dataset?